cmd/monaco/download/download_configs.go

```diff
@@ -109,10 +109,6 @@ func (d DefaultCommand) DownloadConfigsBasedOnManifest(fs afero.Fs, cmdOptions d
 
 	printUploadToSameEnvironmentWarning(env)
 
-	if !cmdOptions.forceOverwrite {
```

Comment:
note that this was appending the environment name to the project, so `project_environment` not `project_project` as you described in the PR description. 

We shortly discussed on Monday to drop this (@Laubi may have created an extra bug/story?).
The only use I can see is if we allow downloading from several environments in the manifest we'd need different project names to identity each env. But we currently don't do that anyway

Reply:
yep that's BS and was a wrong assumption from my side, will drop that commit